lib/printf/printf.cpp

```diff
@@ -33,10 +33,20 @@
 
 #define PRINTF_FLOAT_DECIMALS 3
 
+namespace nanoprintf
+{
 typedef void (*putcf) (void*,char);
 static putcf stdout_putf;
 static void* stdout_putp;
 
+static double abs(double a)

```

Comment:
can we just use `std::abs`

Reply:
I could change it. Should that be in a different PR?

Reply:
meh, sure, we can wait